### ISSUE TITLE
Editorial Meeting Agenda 29 June 2017

### ISSUE THREAD

**Issue by acrymble** [programminghistorian/jekyll#475]
Suggested items for the next editorial meeting. Please add to this anything you'd like to discuss:

~~1) Managing Editor (Jeri) #477 - Jeri to email group with role description.~~
~~2) Editorial Workflow / Capacity (Jeri) #460 #461 #472~~ 
~~3) Website development updates (Fred) #474 #437 - goal is to be finished by end of 7 July.~~
~~4) Zotero API lessons (Fred)~~
~~5) Blog post on redesign (Matt) #437 - Amanda has first refusal, but Fred can do if needed~~
~~6) PDF version of record? (Caleb/Adam) #417~~
7) 10 year anniversary (Adam) [postponed]

**Comment by mdlincoln** [comment]
FYI, the most recent updates to the `redesign` branch can now be previewed at <http://ph-redesign-preview.netlify.com>

**Comment by mdlincoln** [comment]
And remaining redesign issues should all be listed here: https://github.com/programminghistorian/jekyll/milestone/1

**Comment by wcaleb** [comment]
Also #417 to decide if we still want PDF versions of record and what that should look like.

**Comment by acrymble** [comment]
I've been having massive internet issues the past couple of weeks. Is anyone able to lead on the call? I'm worried it will drop constantly if I do it.

**Comment by arojascastro** [comment]
Is there @mdlincoln anything that the Spanish team needs to translate and that we have overlookd? Please let us know.

**Comment by mdlincoln** [comment]
@arojascastro We had been waiting on a translation of the new editor instructions for checking Travis CI results (#425) However, I now think that should wait until after the merge, because there have been many modifications to `es/guia-editor.md` during the redesign. I will notify you when it's ready for translation.

**Comment by wcaleb** [comment]
@acrymble I can take the lead on the call if you can get everyone connected at the beginning (or send me everyone's Skype credentials so that I can)

**Comment by mdlincoln** [comment]
#437 blog posts on the site redesign and technical changes (Matt, Amanda?, Fred?)

**Comment by wcaleb** [comment]
#470 - Fred will complete the inline documentation for the layouts, at which point that will be done.

**Comment by wcaleb** [comment]
We set a deadline for the redesign milestone of July 7: https://github.com/programminghistorian/jekyll/milestone/1

**Comment by wcaleb** [comment]
Caleb to close #417 and make new ticket summarizing discussion of it later this afternoon.
